### DISCUSSION TITLE
общее обсуждение #5 // 2024

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#548] [General]
..

**Comment by Kor-Angar** [comment]
мы правили голоса у Торлофа. и оказалось не правильно. авторы специально не давали пойнты за победу над Родом и Бастером.

**Reply by Kor-Angar** [reply]
```D
	else if (Buster_Duell == TRUE)
	{
		AI_Output (self, other, "DIA_Torlof_RUF_01_17"); 
		if (Buster.aivar[AIV_DefeatedByPlayer] == FALSE)
		{
			AI_Output (self, other, "DIA_Torlof_RUF_01_18");
		};
		//### Point bei Duell ###
	}
```

**Reply by Kor-Angar** [reply]
```D
	else if (Rod.aivar[AIV_DefeatedByPlayer] == TRUE)
	|| 		(Rod_WetteGewonnen == TRUE)
	{
		AI_Output (self, other, "DIA_Torlof_RUF_01_11");
		if (Rod.aivar[AIV_DefeatedByPlayer] == FALSE)
		{
			Points_Sld = Points_Sld + 1; //### sonst Point bei Duell ###
		};
	}
```

**Reply by Kor-Angar** [reply]
а ниже в MDK общая арифметика

> 	//geдndert M.F.					//(Maximal 11 - 13 / insgesamt 6 + 3+1 oder 5+1 + 1 fьr Duell)

### insgesamt 6
1) Волк
2) Jarvis
3) Cord
4) Cipher
5) Sentenza(Деньги)
6) Rod(спор)

###  1 fьr Duell
за дуэли

### 3+1 oder 5+1
квесты Торлофа

***

"проходной" бал это 9. то есть можно "провалить" спор с Родом и не выполнить квест Jarvis, при этом не убивая никого из именных наёмников :-)

**Reply by dosinabox** [reply]
Не понятно. И Род и Бастер вслух говорят что отдают свой голос или "не против":
```c++
AI_Output(self,other,"DIA_Rod_WannaJoin_06_06");	//Если тебе интересно мое мнение, то я не против.
AI_Output(self,other,"DIA_Buster_WannaJoin_13_03");	//Мой голос не много значит, так как здесь я не очень давно, но если Ли спросит, я проголосую за тебя.
```
Потом это вслух проговаривает и Торлоф:
```c++
AI_Output(self,other,"DIA_Torlof_RUF_01_11");	//Похоже, тебе удалось убедить Рода, что ты достаточно силен.
AI_Output(self,other,"DIA_Torlof_RUF_01_17");	//Бастер говорит, что с тобой все в порядке.
```
При этом их согласия не считаются, хоть и проговариваются.

А Бастер вообще не считается человеком Сильвио и не учитывается в дуэлях для Джарвиса. Т.е. в оригинале получить голос от Бастера можно только убив его.

**Reply by Kor-Angar** [reply]
> Не понятно.

не понятно без комментариев разработчиков. а в комменте прямо говорится что их согласия за победу на дуэле грубо говоря суммируется и превращается в один пойнт.


> При этом их согласия не считаются, хоть и проговариваются.

в итоге же считается, просто иначе. не так прямо как нам казалось.  если говорить про авторов то они прямо указывают максимальный балл 11 - 13  без убийств :

- квесты наёмников(не дуэли) +6
- дуэли+1 
- Испытание Торлофа 3+1 oder 5+1

**Reply by Kor-Angar** [reply]
то есть как минимум тебе голос Бастера надо убрать что бы сохранить баланс и не упрощать вступление в наёмники.

**Reply by dosinabox** [reply]
А что делать с этим? 

>  Т.е. в оригинале получить голос от Бастера можно только убив его.

Для дуэлей он никак не учитывается, Джарвис его не перечисляет.

**Reply by Kor-Angar** [reply]
Есть квеста Ярвиса про то что надо побить людей. и за это мы получаем голос Ярвиса +1.

а есть голос за победу в целом на дуэлях. и там победа над Бастером учитывается.

**Reply by Kor-Angar** [reply]
```D
	// --------------------
	// ------ Duelle ------
	// --------------------
	if (Sld_Duelle_gewonnen >= 3)//geдndert M.F.
	{
		Points_Sld = Points_Sld + 1;
		
		if (Points_Sld >= 9)//geдndert M.F.
		{
			AI_Output (self, other, "DIA_Torlof_RUF_01_27");
			AI_Output (self, other, "DIA_Torlof_RUF_01_28");
		}
		else
		{
			AI_Output (self, other, "DIA_Torlof_RUF_01_29");
		};
	};
```

**Reply by dosinabox** [reply]
> Есть квеста Ярвиса про то что надо побить людей. и за это мы получаем голос Ярвиса +1.

Бастер там не учитывается. К людям Сильвио он не относится.

```c++
AI_Output(other,self,"DIA_Jarvis_SylviosMen_15_00");	//Кто из наемников люди Сильвио?
AI_Output(self,other,"DIA_Jarvis_SylviosMen_04_01");	//Их шестеро. Прежде всего, сам Сильвио и его правая рука Буллко.
AI_Output(self,other,"DIA_Jarvis_SylviosMen_04_02");	//Также есть Род, Сентенза, Фестер и Рауль.
```

**Reply by Kor-Angar** [reply]
я пишу про то что есть дуэли , а есть квест Ярвиса. Это разное. Я не пишу про квест Ярвиса и дубасение людей в рамках него(откровенно говоря это самый сложный квест и я его вообще не прохожу нейтралом).

Победа над Бастером и Родом засчитывается как `Sld_Duelle_gewonnen +=1` . 
При `Sld_Duelle_gewonnen >= 3` начисляется голос. Такая ситуация по Бастеру и Роду отображена в комментариях разработчиков. 

**Бастер**
```D
else if (Buster_Duell == TRUE)
	{
		AI_Output (self, other, "DIA_Torlof_RUF_01_17"); 
		if (Buster.aivar[AIV_DefeatedByPlayer] == FALSE)
		{
			AI_Output (self, other, "DIA_Torlof_RUF_01_18");
		};
		//### Point bei Duell ###
	}
```

**Род**
```D
	else if (Rod.aivar[AIV_DefeatedByPlayer] == TRUE)
	|| 		(Rod_WetteGewonnen == TRUE)
	{
		AI_Output (self, other, "DIA_Torlof_RUF_01_11");
		if (Rod.aivar[AIV_DefeatedByPlayer] == FALSE)
		{
			Points_Sld = Points_Sld + 1; //### sonst Point bei Duell ###
		};
	}
```

**Comment by Kor-Angar** [comment]
@dosinabox  не могу найти топик про книги. где то обсуждали в каких стойках можно выводить тексты из готики 1

**Comment by dosinabox** [comment]
![Screenshot_4](https://github.com/user-attachments/assets/6c8b754d-3b33-4cce-b9c9-b909995c5e0d)
Урон: 30
Сила: 10
Длина: 70
Цена: 50

Для сравнения палка с гвоздями с трупа Браго:
Урон: 25
Сила: 10
Длина: 60
Цена: 50

Или бесплатный волчий нож от Кавалорна:
Урон: 30 (+3% бонус)
Сила: 10
Длина: 30
Цена: 350
